tests/TestCase/ORM/CompositeKeysTest.php

```diff
@@ -229,6 +229,7 @@ public function testBelongsToManyEager($strategy)
             'targetTable' => $tags,
             'propertyName' => 'tags',
             'through' => 'SiteArticlesTags',
+            'sort' => 'SiteTags.id',
```

Comment:
you should use an array here in the for of `['SiteTags.id' => 'DESC']` or use call the `order()` function in the test that was failing for you